dm/config/subtask.go

```diff
@@ -44,13 +44,44 @@ var (
 	defaultMaxAllowedPacket = 64 * 1024 * 1024 // 64MiB, equal to TiDB's default
 )
 
+// RawDBConfig contains some low level database config
+type RawDBConfig struct {
+	MaxIdleConns int
+	ReadTimeout  string
+	WriteTimeout string
+}
+
+// DefaultRawDBConfig returns a default raw database config
+func DefaultRawDBConfig(readTimeout string) *RawDBConfig {
+	return &RawDBConfig{
+		ReadTimeout: readTimeout,
```

Comment:
should we need to give `MaxIdleConns` a default value rather than `0`?

Reply:
how about set default to `2`

Reply:
how about set default to `-1`?